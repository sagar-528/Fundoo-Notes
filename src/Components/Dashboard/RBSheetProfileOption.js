import React, { Component } from 'react'
import {View} from 'react-native'
import {Button} from 'react-native-paper'
import RBSheetProfileOptionStyle from '../../Styles/RBSheetProfileOption'

export class RBSheetProfileOption extends Component {

    constructor(props) {
        super(props)
    }
    
    render() {
        return (
            <View>
                <View style = {RBSheetProfileOptionStyle.button_view_style}>
                    <Button
                        onPress = {this.props.takePhoto}
                        style = {RBSheetProfileOptionStyle.button_style}
                        color = 'black'>
                            Take Photo
                    </Button>
                </View>
                <View style = {RBSheetProfileOptionStyle.button_view_style}>
                    <Button
                        onPress = {this.props.chooseFromLibrary}
                        style = {RBSheetProfileOptionStyle.button_style}
                        color = 'black'>
                            Choose From Library
                    </Button>
                </View>
                <View style = {RBSheetProfileOptionStyle.button_view_style}>
                    <Button
                        onPress = {this.props.cancel}
                        style = {RBSheetProfileOptionStyle.button_style}
                        color = 'black'>
                            Cancel
                    </Button>
                </View>
            </View>   
        )
    }
}

export default RBSheetProfileOption
