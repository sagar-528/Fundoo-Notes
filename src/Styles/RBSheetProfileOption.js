import { StyleSheet } from 'react-native'

const RBSheetProfileOptionStyle = StyleSheet.create({
    button_view_style : {
        width : 300, 
        alignSelf : 'center',
        marginBottom : 20
    },

    button_style : {
        backgroundColor : '#339EFF', 
        borderRadius : 20
    }
})