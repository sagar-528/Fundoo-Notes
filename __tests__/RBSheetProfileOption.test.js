import React from 'react';
import {shallow} from 'enzyme';
import RBSheetProfileOption from '../src/Components/Dashboard/RBSheetProfileOption'
import {Button} from 'react-native-paper'

describe('test RBSheetProfileOption', () => {

    it('should match to snapshot', () => {
        const component = shallow(<RBSheetProfileOption />)
        expect(component).toMatchSnapshot();
    })

    it('test component in RBSheetProfileOption component', () => {
        const component = shallow(<RBSheetProfileOption/>)
        expect(component.find(Button)).toHaveLength(3)
    })

})