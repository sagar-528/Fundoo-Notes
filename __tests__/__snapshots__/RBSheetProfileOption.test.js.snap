// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test RBSheetProfileOption should match to snapshot 1`] = `ShallowWrapper {}`;
